docs/dev.rst

```diff
@@ -720,8 +720,9 @@ You can also use it with ``fetch_crashids``:
 
    app@socorro:/app$ socorro-cmd fetch_crashids --num=1 | bin/process_crashes.sh
 
-After running ``bin/process_crashes.sh``, you will need to run the
-processor which will do the actual processing.
+After running ``bin/process_crashes.sh``, you will need to :ref:`run the processor <run-the-processor-locally>` which will do the actual processing.
+
+If you want to view the processed crash, you will need to :ref:`run the webapp <run-the-webapp-locally>`.
```

Comment:
You need to run both the processor and webapp since the processor uses the `VersionString` API. Instead of this, maybe we should say something like:

> Run the processor and webapp with ``make run`` to process the crash reports.

Reply:
Ah okay this is a crash-stats API that the processor uses. Sure I can change it thanks.